examples/twittermap/web/public/javascripts/map/countmap.js

```diff
@@ -2,6 +2,208 @@ angular.module('cloudberry.map')
   .controller('countMapCtrl', function($scope, $compile, cloudberry, cloudberryConfig, MapResultCache,
                                        TimeSeriesCache, moduleManager, cloudberryClient, queryUtil) {
 
+    // Array to store the data for chart
+    $scope.chartData=[];
+    // Map to store the chart data for every polygon
+    $scope.ChartDataMap = new HashMap();
```

Comment:
Let's be consistent on the naming convention as well, like you use first letter small case in the above line.